### PR TITLE
[wasm] Add aot+netcore support.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

**Comment by vargaz** [comment]
@monojenkins build failed

File: mono/mini/interp/transform.c [hunk 1]
```diff
@@ -1845,6 +1845,11 @@ interp_handle_intrinsics (TransformData *td, MonoMethod *target_method, MonoClas
 				*op = MINT_CEQ_I4;
 			}
 		}
+	} else if (in_corlib &&
+			   !strcmp ("System.Runtime.CompilerServices", klass_name_space) &&
+			   !strcmp ("RuntimeFeature", klass_name)) {
+		if (!strcmp (tm, "get_IsDynamicCodeSupported") || !strcmp (tm, "get_IsDynamicCodeCompiled"))
```
Comment (marek-safar) [inline mono/mini/interp/transform.c]: Presumably, IsDynamicCodeCompiled should be 0

Comment (vargaz) [reply mono/mini/interp/transform.c]: What does IsDynamicCodeCompiled mean ? Its not used anywhere right now.

Comment (marek-safar) [reply mono/mini/interp/transform.c]: It's public API so you cannot say it's unused. The intention is to return true when generated code can produce faster code than precompiled version (like regex)

**Comment by vargaz** [comment]
@monojenkins build failed

**Pull request merged**